Makefile

```diff
@@ -10,7 +10,7 @@ post:
 	git push
 
 refresh:
-	GIT_TERMINAL_PROMPT=0 git fetch --all
```

Comment:
i think this still needs to run with GIT_TERMINAL_PROMPT=0 otherwise i get the prompts

Reply:
even with the GIT_TERMINAL_PROMPT=0 statement at the git fetch line? (refresh.bash line 7)

https://github.com/diracdeltas/tweets/pull/18/files#diff-f58ec61af8b53d6b453c330fa93ba2551d844972d990f82bc2048fed90ddb835R7

Reply:
somehow it seems that didn't work